torch_xla/distributed/xrt_init.py

```diff
@@ -0,0 +1,241 @@
+# Copyright Amazon Web Services and its Affiliates. All Rights Reserved.
+# ==============================================================================
+
+import os
+import socket
+import subprocess
+import torch.distributed as dist
+import torch_xla.core.xla_model as xm
+import torch_xla.core.xla_env_vars as xenv
+from torch_xla.utils.utils import get_free_tcp_ports
+from torch_xla.distributed.xla_multiprocessing import _get_devices_per_worker
+
+_NODE_LIST = None
```

Comment:
`_NODE_LIST` looks unused, and `_TCP_STORE` and `_INIT_XRT_ALREADY_CALLED` are already marked `global`, so they don't need to be declared here

Reply:
yes, worked on this and other modifications suggested by you. Resolving the final issue mentioned above related to starting of server processes.